user/gmid/files/tmpfiles.conf

```diff
@@ -0,0 +1,4 @@
+# Create gmid state directories
+
+d /var/lib/gmid 0755 _gmid _gmid -
+d /var/gemini 0755 _gmid _gmid -
```

Comment:
surely this should not be owned by gmid (`/var/www` is not either)

should it be created at all? literally nothing seems to reference it other than some examples

Reply:
`/var/lib/gmid` for certs
`/var/gemini` for .gem files that get served
unlike some other gemini servers, gmid does not handle the creation/rotation of certs so owning the directories isn't necessary as no writes ever happen.

maybe both should be owned by root.

the creation of both could also be skipped, im not sure

Reply:
i guess skipping both is the way to go since there'd still be manual setup to engage in anyway by the user, the certs will more realistically go somewhere in the /etc and the content directory could be left up to the user to setup